usvm-ts/src/main/kotlin/org/usvm/machine/expr/TsExprResolver.kt

```diff
@@ -409,14 +408,14 @@ class TsExprResolver(
             doWithState {
                 val method = ctx.scene
                     .projectAndSdkClasses
-                    .flatMap { it.methods }
+                    .flatMap { it.methods + it.ctor }
                     .singleOrNull { it.signature == expr.method }
                     ?: error("Couldn't find a unique method with the signature ${expr.method}")
 
                 pushSortsForArguments(expr.instance, expr.args, localToIdx)
 
                 callStack.push(method, currentStatement)
-                memory.stack.push(args.toTypedArray(), method.localsCount)
+                memory.stack.push(args.toTypedArray(), method.getDeclaredLocals().size)
```

Comment:
In a neighboring PR with statics, `EtsMethod.localsCount` extension is now fixed and has exactly this impl via declared locals. It is better to fix the extension itself than changing all its usages.